### PR TITLE
pythonPackages.robotframework-databaselibrary: init at 1.2.4

### PR DESCRIPTION
**Pull request by talkara**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/robotframework-databaselibrary/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    sha256 = "627d872b3dda6a308a650ac9e676dadedf9c294e4ef70ad207cbb86b78eb8847";
+  };
+
+  # unit tests are impure
+  doCheck = false;
+
+  propagatedBuildInputs = [ robotframework ];
+
+  meta = with stdenv.lib; {
+    description = "Database Library contains utilities meant for Robot Framework";
```
Comment (mmahut) [inline pkgs/development/python-modules/robotframework-databaselibrary/default.nix]: Do you mind adding yourself as a maintainer?

**Comment by mmahut** [comment]
@GrahamcOfBorg build python3Packages.robotframework-databaselibrary python2Packages.robotframework-databaselibrary

File: pkgs/development/python-modules/robotframework-databaselibrary/default.nix (hunk 1 above)
Comment (disassembler) [reply pkgs/development/python-modules/robotframework-databaselibrary/default.nix]: I agree. I'm hesitant to add packages that no one wants to maintain. But if you're willing to be a maintainer for this, I approve.

**Comment by mmahut** [comment]
@GrahamcOfBorg build python3Packages.robotframework-databaselibrary python2Packages.robotframework-databaselibrary

**Pull request merged**